src/terrama2/services/alert/impl/DocumentPDF.hpp

```diff
@@ -75,26 +91,18 @@ namespace terrama2
               else
               {
                 QString errMsg = QObject::tr("PDF document is not implemented for this data series!");
-                TERRAMA2_LOG_ERROR() << errMsg;
                 throw NotifierException() << ErrorDescription(errMsg);
               }
 
               core::replaceReportTags(body, report);
 
-              QDir dir(QString::fromStdString(report->documentSavePath()));
 
-              if(!dir.exists())
-              {
-                QString errMsg = QObject::tr("Couldn't create PDF document: Informed directorie doesn't exist!");
-                TERRAMA2_LOG_ERROR() << errMsg;
-                throw NotifierException() << ErrorDescription(errMsg);
-              }
 
-              std::string path = dir.absolutePath().toStdString() + "/"
-                                 + terrama2::core::simplifyString(report->name())
-                                 + ".pdf";
+              std::string filePath = dir.absolutePath().toStdString() + "/"
+                                     + terrama2::core::simplifyString(report->name())
```

Comment:
I think it's best to use the "document_save_path" as complete path to the file, not just folder.